mini-modeline.el

```diff
@@ -80,8 +80,28 @@ Will be set if `mini-modeline-enhance-visual' is t."
   :type 'boolean
   :group 'mini-modeline)
 
+(defface mini-modeline-active-window-modeline
+  '((((background light))
+     :background "#55ced1" :height 0.14 :box nil)
+    (t
+     :background "#008b8b" :height 0.14 :box nil))
+  "Modeline face for active window."
+  :group 'mini-modeline)
+
+(defface mini-modeline-inactive-window-modeline
+  '((((background light))
+     :background "#dddddd" :height 0.1 :box nil)
+    (t
+     :background "#333333" :height 0.1 :box nil))
+  "Modeline face for inactive window."
+  :group 'mini-modeline)
+
 (defvar mini-modeline--orig-mode-line mode-line-format)
 (defvar mini-modeline--echo-keystrokes echo-keystrokes)
+(defvar mini-modeline--orig-mode-line-remap
+  (alist-get 'mode-line face-remapping-alist))
```

Comment:
This should be `(or (alist-get 'mode-line face-remapping-alist) 'mode-line)`, so we will correctly restore the original face when disable `mini-modeline`. Same for bellow.

Reply:
Done.